src/agent/onefuzz/src/az_copy.rs

```diff
@@ -29,15 +34,33 @@ impl fmt::Display for Mode {
     }
 }
 
+async fn read_azcopy_log_file(path: &Path) -> Result<String> {
```

Comment:
They're nearby, but as a doc comment for the function, could you briefly note the assumption that `path` is not only the `AZCOPY_LOG_LOCATION` set elsewhere, but a temporary directory created for this purpose?